code/_onclick/item_attack.dm

```diff
@@ -110,4 +110,4 @@
 		visible_message("<span class='warning'>[I] bounces harmlessly off of [src].</span>",\
 					"<span class='warning'>[I] bounces harmlessly off of [src]!</span>")
 	else
-		return ..()
\ No newline at end of file
+		return ..()

```

Comment:
If we had standardised line endings this sort of thing wouldn't happen! I can't fix it without introducing MORE line ending problems at the moment, so I guess I'll close this and try again when my repository is less on fire.